### PR TITLE
chore(theme): refactor getters from composables to theme

### PR DESCRIPTION
**Pull request by youanden**
## Related Issue

https://vsf.atlassian.net/browse/M2-261

Work in progress

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/getters/addressGetter.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import { Countries, Country } from '@vue-storefront/magento-api';
-import { AddressGetter } from '../types';
+import { Countries, Country, AddressGetter } from '@vue-storefront/magento';
```
Comment (sethidden) [inline packages/theme/getters/addressGetter.ts:1]: @Frodigo is taking types from /magento better than /magento-api in this case? We have a separate task for migrating the types into the theme, but just wondering though

File: packages/theme/getters/categoryGetters.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import { CategoryGetters, AgnosticCategoryTree, AgnosticBreadcrumb } from '@vue-storefront/core';
-import { Category } from '@vue-storefront/magento-api';
-import { buildCategoryTree } from '../helpers/buildCategoryTree';
+import { Category, buildCategoryTree } from '@vue-storefront/magento';
```
Comment (sethidden) [inline packages/theme/getters/categoryGetters.ts:2]:
```suggestion
import { Category } from '@vue-storefront/magento';
```
We already have our own buildCategoryTree instance within the `theme` package. Can you please use it instead? They're duplicated for now but I think the package version will be removed soon

File: packages/theme/getters/productGetters.ts [hunk 3]
```diff
@@ -6,13 +6,11 @@ import {
   ProductGetters as ProductGettersBase,
 } from '@vue-storefront/core';
 import {
-  BundleProduct,
+  htmlDecode, BundleProduct,
```
Comment (sethidden) [inline packages/theme/getters/productGetters.ts:9]:
```suggestion
  BundleProduct,
```
Same as in previous comment - please use the `theme` version (import {htmlDecode} from '../helpers/htmlDecoder')

Comment (Frodigo) [reply packages/theme/getters/productGetters.ts:9]: Hmm, I can't find htmlDecoder in theme.

Comment (sethidden) [reply packages/theme/getters/productGetters.ts:9]: Right, misread on my side

**Review by Frodigo — CHANGES_REQUESTED** [review]
@youanden make sure that build pass.

**Review by Frodigo — APPROVED** [review]

**Pull request merged**